hpy/devel/include/cpython/hpy.h

```diff
@@ -21,6 +21,12 @@
 
 #define HPyAPI_FUNC(restype) HPyAPI_STORAGE restype
 
+#ifdef __GNUC__
+#define HPyAPI_RUNTIME_FUNC(restype) __attribute__((visibility("hidden"))) restype
```

Comment:
should this use `_HPy_HIDDEN`?

Reply:
Fix by defining _HPy_HIDDEN more nicely and then defining HPyAPI_RUNTIME_FUNC in terms of _HPy_HIDDEN.